learnosity_sdk/request/init.py

```diff
@@ -245,8 +241,24 @@ def hash_list(self, l):
         "Hash a list by concatenating values with an underscore"
         return hashlib.sha256("_".join(l).encode('utf-8')).hexdigest()
 
-    def disable_telemetry(self):
-        self.__telemetry_enabled = False
+    def add_telemetry_data(self):
+        if self.__telemetry_enabled:
+            if 'meta' in self.request:
+                self.request['meta']['sdk'].update(self.get_sdk_meta())
```

Comment:
I think this would throw an error because `'sdk'` is probably an undefined key. We probably want to do something like this: `self.request['meta']['sdk'] = self.get_sdk_meta()`, which will overwrite the SDK with our new data.